pages/wiki/useful-commands.md

```diff
@@ -5,6 +5,8 @@ layout: documentation
 
 AsteroidOS comes with several command line tools that are very handy to control a watch from SSH or a script. This page tries to document the basic features of those commands, don't hesitate to use their --help options if you want to learn more.
 
+Input commands and gestures to enter the fastboot bootloader on your watch manually, are listed in the last section of this page. 
```

Comment:
fastboot is a mode of the bootloader. I suggest changing the sentence to:
"Commands and gestures to enter the fastboot (bootloader) mode are at the bottom of this page."